spec/lib/tenejo/preflight_spec.rb

```diff
@@ -89,15 +102,18 @@
     end
 
     it "has validation" do
-      [:work, :collection, :file].each do |x|
+      FileUtils.touch("tmp/uploads/MN-02 4.png")
+      [:work, :collection].each do |x|
         graph[x].each do |y|
           expect(y.valid?).to eq true
         end
       end
+      expect(graph[:file].first.valid?).to be false
+      expect(graph[:file].last.valid?).to be true
```

Comment:
I'm finding it much easier to follow the explicit examples in line 111 & 112 compared to the loops in 106-110.

Is there any reason not to just check validation on the first collection and work instead of all of them?  (i.e. is this  testing that a validation process exists, or that it passes for everything?)

```
expect(graph[:collection].first.valid?).to be true
expect(graph[:work].first.valid?).to be true
expect(graph[:file].last.valid?).to be true

expect(graph[:file].first.valid?).to be true
```

Also, either way you might want to use `:aggregate_failures` on the example if you're running expectations on multiple things - see https://relishapp.com/rspec/rspec-core/docs/expectation-framework-integration/aggregating-failures#use-%60:aggregate-failures%60-metadata